src/resolve/FieldValues.jl

```diff
@@ -74,7 +74,7 @@ validmax(a::FieldValue) = a.l0 >= 0
 
 # like usual indmax, but favors the highest indices
 # in case of a tie
-function Base.indmax(f::Field)
+function indmax(f::Field)
```

Comment:
`Base.argmax`?

Reply:
Yeah, I just saw that it is supposed to be `Base.argmax` from your PR

Reply:
On second thought, we might as well just not extend the Base function, no reason to.